Task07/script.js

```diff
@@ -0,0 +1,47 @@
+(() => {
+'use strict';
+	
+	//ドロップダウン内の製品と対応する製品コードの定義
+	const dropdownData = {
+		'kintone': 'KN',
+		'Garoon': 'GR',
+		'サイボウズ Office': 'OF',
+		'Mailwise': 'MW'
+	};
+
+	const avoidDuplication = (event) => {
+		//日付からハイフンを抜いた文字列を作成
+		const dateData =() => {
+			const dateStr = event.record.日付.value;	
+			return dateStr.replace(/-/g, '');
+		} ;
```

Comment:
関数化しないで1行で書くとスッキリします！
```suggestion
		const dateStr = event.record.日付.value.replace(/-/g, '');
```
注意：26行目も合わせて要変更

Reply:
なんでも関数化する癖あります笑
高宮さんにもアドバイスいただきましたが、一回しか使わないところは関数化しないことを忘れないようにします。